pkg/venafi/fake/connector.go

```diff
@@ -229,7 +229,10 @@ func (c *Connector) RevokeCertificate(revReq *certificate.RevocationRequest) (er
 }
 
 func (c *Connector) ReadZoneConfiguration() (config *endpoint.ZoneConfiguration, err error) {
-	return endpoint.NewZoneConfiguration(), nil
+	config = endpoint.NewZoneConfiguration()
+	policy, err := c.ReadPolicyConfiguration()
```

Comment:
Should `config` still be returned if `err != nil`?

Reply:
it's fake connection. It never returns error.